x-pack/test/api_integration/apis/security_solution/events.ts

```diff
@@ -454,6 +468,60 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
+    // TODO: unskip this test once authz is added to search strategy
+    it.skip('Make sure that we get Timeline data using the hunter role and do not receive observability alerts', async () => {
```

Comment:
This looks like it should still be skipped in the final version of this PR, sounds like the intent is to do unskip it in a future PR, but I am not 100% sure and I wanted to mention it in case it was an oversight!